src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.cs

```diff
@@ -9586,7 +9586,7 @@ private bool UriEqual(Uri uri1, string uri1Str, string uri2Str, XmlResolver reso
 
         /// <summary>
         /// This method should be called every time the reader is about to consume some number of
-        ///   characters from the input. It will count it agains the security counters and
+        ///   characters from the input. It will count it again. The security counters
```

Comment:
I believe this was supposed to be "count it against the". That should be fixed and the "and" restored.